underscore.js

```diff
@@ -143,6 +143,7 @@
   var MAX_ARRAY_INDEX = Math.pow(2, 53) - 1;
   var getLength = property('length');
   var isArrayLike = function(collection) {
+    if ( typeof collection === 'function' && collection.constructor !== root.NodeList ) return false;
```

Comment:
The [[Class]] for NodeLists is "NodeList", at least for my modern chrome. Would that test be sufficient?